components/sidebar_right/sidebar_right.tsx

```diff
@@ -123,27 +125,20 @@ export default class SidebarRight extends React.PureComponent<Props, State> {
     }
 
     componentDidMount() {
-        window.addEventListener('resize', this.determineTransition);
```

Comment:
these listener is not needed any more since we are not controlling this components open and close with component state, but through prop.